sdk/spring-sdk/src/test/java/kalix/springsdk/testmodels/eventsourcedentity/EventSourcedEntitiesTestModels.java

```diff
@@ -75,6 +75,46 @@ private Integer privateMethodSimilarSignature(Integer event) {
     }
   }
 
+
+  @EntityType("counter")
+  public static class CounterEventSourcedEntityWithEntityKeyOnMethod extends EventSourcedEntity<Integer> {
+    @EntityKey("id")
+    @GetMapping("/eventsourced/{id}/int/{number}")
+    public Integer getInteger(@PathVariable Integer number) {
+      return number;
+    }
+  }
+
+  @EntityKey("id")
+  @EntityType("counter")
+  public static class CounterEventSourcedEntityWithEntityKeyMethodOverride extends EventSourcedEntity<Integer> {
+
+    @EntityKey("counter_id")
```

Comment:
Question is, what should we do when we have EntityKey on type level and on method level? 

I think a sensible default it to let pick the method one and override the type one. But open for other suggestions.

Reply:
Another possibility I can think of would be raising an error, not allowing two EntityKey annotations.

Reply:
It makes sense to me, yup.